app/assets/javascripts/code/controllers/_mobile_sessions_map_ctrl.js

```diff
@@ -41,17 +41,22 @@ export const MobileSessionsMapCtrl = (
         $scope.$digest();
       });
     }
-    _.each(['sensor', 'location', 'usernames'], function(name) {
+
+    ['sensor', 'location', 'usernames', 'layers'].forEach(function(name) {
```

Comment:
Tested and refactored to remove `_`. In the long term I'd like to remove that dependency. I'd prefer to use Ramda or something similar